beacon-chain/core/state/skip_slot_cache.go

```diff
@@ -22,3 +26,14 @@ var (
 		Help: "The total number of cache misses on the skip slot cache.",
 	})
 )
+
+func cacheKey(bState *pb.BeaconState) ([32]byte, error) {
+	// the latest header has a zeroed 32 byte hash as the state root,
+	// which isnt necessary for the purposes of making the cache key. As
+	// the parent root and body root are sufficient to prevent any collisions.
+	blockRoot, err := ssz.HashTreeRoot(bState.LatestBlockHeader)
```

Comment:
is there any advantage on using hash tree root than something like proto hash?

Reply:
Oh whoops sorry @terencechain merged in before seeing the comment, I think it shouldn't make any difference and proto hash could be faster

Reply:
If this is still appearing in the flame graph, we can revisit. I think HTR on this object is less significant, but proto hash might still be a noticeable improvement.